### ISSUE TITLE
[Feature Request] Make chests unbreakable without the key

### ISSUE THREAD

**Issue by pn3m0n1c** [ThatGravyBoat/Ironchests#18]
Hey! I love this mod it's great, but it bothers me a lot that locked chests can be broken with the pickaxe (I know the ones made of better materials take longer to break, but they end up breaking sooner or later), it causes the locks to be practically useless. 

I think if you could set a condition like: "If the state of the chest is locked, the chest is unbreakable." this problem would be easily fixed. Thank you very much for this mod we enjoyed it very much on our server!

I tried to change a JSON parameter I found from the chests in the code that is uploaded (I'm a complete newbie to coding minecraft mods, sorry if I say something stupid, I suspect the mod code isn't really that because I don't see any functions or classes or anything like that) called "destroyTime" to make them destroy extremely slowly, but when I changed it and tried to export to .jar minecraft stopped recognizing it as a mod.

**Comment by ThatGravyBoat** [comment]
if you are talking about the JSON API here is the reference for destroy time: https://thatgravyboat.gitbook.io/iron-chests-restocked-wiki/v/chests/property/builder#destroy-time-destorytime-float

**Comment by pn3m0n1c** [comment]
Yes exactly, it's just that parameter that I changed, I changed those values to some like 500, 999, 500000... and it doesn't seem to have any effect, it doesn't matter if I put 500 or 500000, there seems to be a maximum time. In fact I have set the same destruction time for both the copper and the netherite chest, and it turns out that the copper one breaks before even though it has the same values.

![ezgif com-gif-maker](https://user-images.githubusercontent.com/92324278/188478685-8b7e0c7e-7e3a-40de-8b22-18ce5af164ff.gif)

Thank you for your response!

**Comment by ThatGravyBoat** [comment]
well I cant do much about it not having much of an effect as that is just telling mojang that's the destroy time.
